Tests/UnitTests/Networking/HTTPResponseTests.swift

```diff
@@ -31,7 +31,7 @@ class ErrorResponseTests: XCTestCase {
 
         expect(error.domain) == ErrorCode.errorDomain
         expect(error.code) == ErrorCode.invalidCredentialsError.rawValue
-        expect(error.userInfo[Backend.RCAttributeErrorsKey]).to(beNil())
+        expect(error.userInfo[ErrorDetails.attributeErrorsKey as String]).to(beNil())
```

Comment:
maybe we can have an extension so that we can do `ErrorDetails.attributeErrorsKey.string` directly instead of doing the cast everywhere?

Reply:
Yeah I thought about something like that.
How about:

```swift
extension NSError.UserInfoKey {
  static let attributeErrors = "xxx"
  // ... etc
}

enum ErrorDetails {
  static let attributeErrorsKey = NSError.UserInfoKey.attributeErrors as String
  // ... etc
}
```

So when we need the key we can just do `.attributeErrors`, but if not `ErrorDetails.attributeErrors` just works?

Reply:
Changed